### PR TITLE
Attribute Rule examples

### PR DESCRIPTION
**Pull request by MikeMillerGIS**
These were crafted and some shipped with the Utility Network solution configurations.

### CODE REVIEW

File: calculation/CopyValueIntersectingFeature.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+```js
+// This rule will populate the edited features field with a value from an intersecting feature
+
+// Value to copy from the intersected feature
+var intersecting_field = "maintby";
+//Field rule is assigned to in the Attribute Rule
+var feature_field = "maintby";
+
+// Create feature set to the intersecting class using the GDB Name
+var intersecting_featset = FeatureSetByName($datastore, 'WaterLine', [intersecting_field], true);
```
Comment (ekenes) [inline calculation/CopyValueIntersectingFeature.md]: I was under the impression that passing vars for field names as opposed to a string literal causes issues in Pro. This  expression is good as far as I'm concerned, but I just wanted to make sure this works as expected.

Comment (MikeMillerGIS) [reply calculation/CopyValueIntersectingFeature.md]: Variables for FeatureClass does not work, variables for fields does.  Let me build a sample FGDB like I did for the other rule so we can verify

Comment (MikeMillerGIS) [reply calculation/CopyValueIntersectingFeature.md]: Want me to move to a Attribute Rule Calculation folder?

Comment (MikeMillerGIS) [reply calculation/CopyValueIntersectingFeature.md]: Verified and added sample, added a few more from our sewer config too

Comment (ekenes) [reply calculation/CopyValueIntersectingFeature.md]: > Want me to move to a Attribute Rule Calculation folder?

That would be great!

Comment (MikeMillerGIS) [reply calculation/CopyValueIntersectingFeature.md]: Moved to own folder and updated readme

**Pull request merged**